@xen-orchestra/fs/src/fs.spec.js

```diff
@@ -350,5 +350,35 @@ handlers.forEach(url => {
         }
       )
     })
+
+    describe('#truncate()', () => {
+      forOwn(
+        {
+          'inbound truncates': (() => {
+            const length = random(0, TEST_DATA_LEN)
+
+            const expected = Buffer.alloc(length)
+            TEST_DATA.copy(expected)
+
+            return { length, expected }
+          })(),
+          'outbound truncates': (() => {
+            const length = random(TEST_DATA_LEN + 1, TEST_DATA_LEN * 2)
```

Comment:
Why `TEST_DATA_LEN + 1` and not `TEST_DATA_LEN`?

Reply:
Just to be sure that we will have a bigger length value than the original one.